usd/src/cmd/sdf2usd.cc

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <string.h>
+
+#include <ignition/utils/cli/CLI.hpp>
+#include <pxr/usd/usd/stage.h>
+
+#include "sdf/sdf.hh"
+#include "sdf/usd/World.hh"
+
+//////////////////////////////////////////////////
+/// \brief Enumeration of available commands
+enum class Command
+{
+  kNone,
+};
+
+//////////////////////////////////////////////////
+/// \brief Structure to hold all available topic options
+struct Options
+{
+  /// \brief Command to execute
+  Command command{Command::kNone};
+
+  /// \brief input filename
+  std::string inputFilename{"input.sdf"};
+
+  /// \brief output filename
+  std::string outputFilename{"output.sdf"};
```

Comment:
Should the output file name have a `.usda` suffix instead of `.sdf`, since we are converting to usd here?